sample.env

```diff
@@ -13,9 +13,9 @@ DOCKER_HOST_IP=
 
 WEBAPP_HOST_IP=${DOCKER_HOST_IP}
 # Note if using KEYCLOAK_DOMAIN for the docker-compose-webapp-deployment.yml file you will need to include http:// or https://
-KEYCLOAK_DOMAIN=cvmanager.auth.com
+KEYCLOAK_DOMAIN=http://cvmanager.auth.com:8084
 KC_HOST_IP=${DOCKER_HOST_IP}
-WEBAPP_DOMAIN=cvmanager.local.com
+WEBAPP_DOMAIN=http://cvmanager.local.com:8081
```

Comment:
This doesn't actually seem to work for me. The webapp should actually be deployed on port 80 and the CV Manager API at port 8081. Keycloak tries to redirect to this domain at port 8081 and fails with it configured in this manner. We would need to make a new domain variable for the API if we wanted to keep the protocol and port in the ENV file.